examples/forward/tesseroid.py

```diff
@@ -1,6 +1,6 @@
 """
-Tesseroid Forward Modelling
-===========================
+Single Tesseroid
```

Comment:
```suggestion
Tesseroids (spherical prisms)
```

Reply:
Yes, I like it. I was thinking that in the future other tesseroids examples may appear on the gallery, such as Tesseroid Mesh. But I think the difference between titles would be on that "Meshes", so the "Single" part on this one is irrelevant. And adding "spherical prisms" is nice, I tend to forget no everyone is familiar with the tesseroid term.